### DISCUSSION TITLE
Git Large File not working

### DISCUSSION THREAD

**Discussion by dji627** [git-lfs/git-lfs#5517] [Q&A]
I followed all the steps on  https://git-lfs.github.com to configure the Large File Storage but I'm still getting this message:
![image](https://github.com/git-lfs/git-lfs/assets/73142123/890eb4fc-6c3a-4874-bcea-829e3324eaec)

**Comment by bk2204** [comment]
Hey,

What to do in this case [is outlined in the FAQ](https://github.com/git-lfs/git-lfs/blob/main/docs/man/git-lfs-faq.adoc). If you're still having problems after that, let us know and provide more details of what you've done to get to this point (e.g., is the repository already existing and were the files added before `git lfs track`?), and we can provide more assistance.
